pkg/provider/azure.go

```diff
@@ -1389,3 +1422,13 @@ func isNodeReady(node *v1.Node) bool {
 	}
 	return false
 }
+
+func (az *Cloud) getActiveNodesByLoadBalancerName(lbName string) sets.Set[string] {
+	for _, multiSLBConfig := range az.MultipleStandardLoadBalancerConfigurations {
+		if strings.EqualFold(strings.TrimSuffix(lbName, consts.InternalLoadBalancerNameSuffix), multiSLBConfig.Name) {
+			return multiSLBConfig.ActiveNodes
```

Comment:
please exclude unmanaged nodes

Reply:
unmanaged nodes are also included in `nodes []*v1.Node` object, so will be included here as well.